### DISCUSSION TITLE
Price Multiplied by 100?

### DISCUSSION THREAD

**Discussion by livingmine** [medusajs/medusa#4031] [Q&A] [answered]
From the docs,

> By default, prices in Medusa are stored as the price entered by the admin multiplied by a 100.

super curious why is it the case?

**Comment by olivermrbl** [comment, marked as answer]
@livingmine – thanks for submitting this question. It made it clear that the concept of prices was not sufficiently covered in our documentation. We've added more details on the topic [here](https://github.com/medusajs/medusa/pull/4035/files). Let me know if it answers your question.

**Reply by tsvetann** [reply]
@olivermrbl at first this sounds unintuitive. Is there a reason why this price presentation isn't handled on application level? Thanks
